robot/Cumulus/tests/browser/gift_entry/review_donations_update_pmt_bge.robot

```diff
@@ -0,0 +1,93 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/GiftEntryPageObject.py
+...             robot/Cumulus/resources/OpportunityPageObject.py
+...             robot/Cumulus/resources/PaymentPageObject.py
+...             robot/Cumulus/resources/AccountPageObject.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             API Check And Enable Gift Entry
+...             Setup Test Data
+Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+
+*** Keywords ***
+Setup Test Data
+    [Documentation]      Creates the organiation account,opportunity and queries the payment record required for the test
+    ...                  along with getting dates and namespace required for test.
+    &{account} =         API Create Organization Account    Name=${faker.company()}
+    Set suite variable   &{ACCOUNT}
+    ${fut_date} =            Get Current Date         result_format=%Y-%m-%d    increment=2 days
+    Set suite variable   ${FUT_DATE}
+    ${cur_date} =            Get Current Date         result_format=%Y-%m-%d    
```

Comment:
Can you make the variable name to Uppercase during assignment too, to be consistent
     &{ACCOUNT} =         API Create Organization Account    Name=${faker.company()}
    Set suite variable   &{ACCOUNT}

Reply:
done